src/main/scala/io/iohk/ethereum/jsonrpc/EthService.scala

```diff
@@ -312,6 +323,60 @@ class EthService(
     }
   }
 
+  def getBalance(req: GetBalanceRequest): ServiceResponse[GetBalanceResponse] = {
+    Future.successful {
+      withAccount(Address(req.address), req.block) { account =>
+        GetBalanceResponse(account.balance)
+      }
+    }
+  }
+
+  def getStorageAt(req: GetStorageAtRequest): ServiceResponse[GetStorageAtResponse] = {
+    import io.iohk.ethereum.rlp.UInt256RLPImplicits._
+
+    val byteArrayUInt256Serializer = new ByteArrayEncoder[UInt256] {
+      override def toBytes(input: UInt256): Array[Byte] = input.bytes.toArray[Byte]
+    }
+
+    val rlpUInt256Serializer = new ByteArraySerializable[UInt256] {
```

Comment:
Then we make the `domain` package dependent on `mpt` which, imo, is even worse. I think that should be a third layer. But for now I'm fine with either approach as long those serializers are easy to move (single file).

@mcsherrylabs @AlanVerbner @ntallar as this possibly relates to the ongoing refactoring effort (beyond `network`) maybe you'd like to voice your opinion?